CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Backtrace Cocoa Release Notes
 
+## Version 1.7.4-beta1
+- Modify CI job to run tests daily on schedule in #81
+- fixing broken symbolification link in #80
```

Comment:
The change for [PR#80](https://github.com/backtrace-labs/backtrace-cocoa/pull/80) is now obsolete (the link was removed) since the README has been updated. Should it be removed from the changelog?